packages/connect-wallet-modal/src/components/Ledger/LedgerAccountButton.tsx

```diff
@@ -0,0 +1,113 @@
+import React, { FC } from 'react';
+import { Address, Text, useBreakpoint } from '@lidofinance/lido-ui';
+import styled, { keyframes } from '@reef-knot/ui-react/styled-wrapper';
+import { LedgerWalletIcon } from './icons/LedgerWalletIcon';
+import { AccountRecord } from './types';
+
+const ButtonStyled = styled.div`
+  width: 100%;
+  background-color: var(--lido-color-backgroundSecondary);
+  border-radius: 10px;
+  display: flex;
+  padding: 16px 20px;
+  justify-content: space-between;
+  align-items: center;
+
+  margin-top: 10px;
+
+  :first-child {
+    margin-top: 0;
+  }
+
+  :hover {
+    cursor: pointer;
+    background-color: var(--lido-color-backgroundDarken);
+  }
+`;
+
+const AddressContainerStyled = styled.div`
+  display: flex;
+  align-items: center;
+  margin-right: 10px;
+`;
+
+const IconContainerStyled = styled.div`
+  margin-right: 20px;
+  
+  ${({ theme }) => theme.mediaQueries.md} {
+    margin-right: 10px;
+  }
+`;
+
+const gradientKeyframes = keyframes`
+  100% {
+    background-position: 368px 0;
+  }
+`;
+
+const SkeletonWrapper = styled(ButtonStyled)`
+  background: linear-gradient(
+      270deg,
+      var(--lido-color-backgroundSecondary) 0%,
+      var(--lido-color-backgroundSecondary) 0.01%,
```

Comment:
looks strange, you sure this is intended?

Reply:
fixed here: https://github.com/lidofinance/reef-knot/pull/85/commits/bf38684fd0901ad3ba3a70555597283aa09f401f